### DISCUSSION TITLE
自定义字体设置放在main.tex的哪一行？问题如图，无法编译

### DISCUSSION THREAD

**Discussion by ChenhangXu** [sjtug/SJTUThesis#1017] [提问] [answered]
![image](https://github.com/sjtug/SJTUThesis/assets/131256629/95cfa00f-3b6b-4104-a9a9-59ecba485925)

**Comment by LogCreative** [comment]
请提供报错信息。可以先检查一下字体文件是否完整、有没有损坏。

**Reply by ChenhangXu** [reply]
![image](https://github.com/sjtug/SJTUThesis/assets/131256629/fcdf830f-f247-46f2-8080-93142275eec4)

**Reply by LogCreative** [reply, marked as answer]
检查是否使用 XeLaTeX 编译器进行编译。

**Reply by ChenhangXu** [reply]
okay，解决了。之前用的在线版本，是XeLaTeX编译器。后来总是compile timeout,换成交大的overleaf就没注意这个问题了
